README.md

```diff
@@ -143,3 +143,24 @@ func main() {
 
 The main examples have been moved - you can find them in their [own repository](https://github.com/fyne-io/examples/).
 
+## Non-standard library dependencies
+
+Fyne requires the following imports which are not included in the go standard library:
+
+* github.com/go-gl/gl/v3.2-core/gl
+* github.com/go-gl/glfw/v3.2/glfw
+* github.com/golang/freetype
+* github.com/srwiley/oksvg
+* github.com/srwiley/rasterx
+* golang.org/x/image/font
+
+These can be included in your gopath by the following 'get' commands:
+
+* "go get github.com/go-gl/gl/v3.2-core/gl"
```

Comment:
This is not necessary. A `go get github.com/fyne-io/fyne/...` would pull in all of these

Reply:
In my experience it has never pulled in these dependencies

Reply:
Was that back when you needed to pass -tags gl? Since we moved this to the default driver it should indeed be downloading the dependencies at the same time.

I will try on a fresh installation and see what happens.

Reply:
You're right it does not work.
However, instead of listing the dependencies how about we do this instead? (as it will always work, even when the deps change)

```bash
go get -u github.com/fyne-io/fyne
cd $GOPATH/src/github.com/fyne-io/fyne
go get -u ./...
```